### PR TITLE
Dji matrice 210 rtk

### PR DESCRIPTION
**Pull request by ben-isaac**
Modified pelican urdf file to include the DJI Matrice 210 visuals with 4 stationary propellers.

### CODE REVIEW

File: rotors_description/urdf/pelican.xacro [hunk 1]
```diff
@@ -25,10 +22,10 @@
   <xacro:property name="namespace" value="$(arg namespace)" />
   <xacro:property name="rotor_velocity_slowdown_sim" value="10" />
   <xacro:property name="use_mesh_file" value="true" />
-  <xacro:property name="mesh_file" value="package://rotors_description/meshes/pelican.dae" />
+  <xacro:property name="mesh_file" value="package://rotors_description/meshes/dji_matrice_210_v3_chassis.dae" />
   <xacro:property name="mass" value="1.0" /> <!-- [kg] -->
   <xacro:property name="body_width" value="0.1" /> <!-- [m] -->
-  <xacro:property name="body_height" value="0.2" /> <!-- [m] -->
+  <xacro:property name="body_height" value="0.54" /> <!-- [m] -->
```
Comment (archielee) [inline rotors_description/urdf/pelican.xacro:28]: Just wondering why we need to change body height

File: rotors_description/urdf/dji_matrice_210_v1.xacro [hunk 2]
```diff
@@ -0,0 +1,242 @@
+<?xml version="1.0"?>
```
Comment (archielee) [inline rotors_description/urdf/dji_matrice_210_v1.xacro:1]: We will eventually switch to this xacro, correct? And undo the Pelican changes

Comment (ben-isaac) [reply rotors_description/urdf/dji_matrice_210_v1.xacro:1]: Yes

Comment (ben-isaac) [reply rotors_description/urdf/dji_matrice_210_v1.xacro:1]: This is a temporary workaround. I added an offset of 0.268m in the spawn file but the drone still was launched with its landing gear through the ground plane. Increasing the box height seems to fix it though I preferred not changing it if possible. When we add the next revision of the DJI Matrice 210 URDF we will use the actual height and width corresponding to that of the DJI quadrotor.

**Comment by archielee** [comment]
We'll need to make sure we revert the pelican changes when we move to a true Matrice model

**Pull request merged**